### PR TITLE
12392: create change in city map section on landuse-form

### PR DESCRIPTION
**Pull request by trbmcginnis**
**Overall Description**
Add new "Change in City Map" section to the landuse-form. 

**Part of Larger Feature**
Land Use Form

**Technical Description**
- addition of checkboxes and radio buttons in new component that is rendered on the landuse-form

NOTE: employs the YES_NO_INTEGER optionset that @godfreyyeung created "to accommodate the unfortunate Yes/No options which map to 1 and 0, instead of true and false -- uses native Ember Input component to render checkboxes tied to integer values, since our Checkbox component assumes boolean values"

Closes [AB#12392](https://dcp-paperless.visualstudio.com/d36fd830-9029-4b77-b0c4-b0df2012eb98/_workitems/edit/12392)

### CODE REVIEW

File: client/app/components/packages/landuse-form/change-city-map.hbs [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+        </label>
+
+        <br>
+
+        <form.Field
+          @attribute="dcpEasement3"
+          @type="checkbox"
+          as |Checkbox|
+        >
+          <Checkbox>
```
Comment (godfreyyeung) [inline client/app/components/packages/landuse-form/change-city-map.hbs:297]: Is this intentional?

File: client/app/components/packages/landuse-form/change-city-map.hbs [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+        data-test-related-acquisition-radio-group
+        as |Q|
+      >
+        <Q.Legend>
+          Is there a related disposition of property?
+        </Q.Legend>
+
+          <form.Field
+            @attribute="dcpRelatedacquisition"
+            @type="radio"
```
Comment (godfreyyeung) [inline client/app/components/packages/landuse-form/change-city-map.hbs:331]: Just curious -- can you use the radio-group type here instead?

File: client/app/components/packages/landuse-form/change-city-map.hbs (hunk 1 above)
Comment (godfreyyeung) [reply client/app/components/packages/landuse-form/change-city-map.hbs:297]: Oh.. does easement map to true/false unlike the others? CRM 🤣

File: client/app/helpers/optionset.js [hunk 3]
```diff
@@ -81,6 +81,19 @@ const OPTIONSET_LOOKUP = {
     dcpNewfacilityopt: COMMON_OPTIONSETS.YES_NO_INTEGER,
     dcpIsprojectlistedinstatementofneedsopt: COMMON_OPTIONSETS.YES_NO_INTEGER,
     dcpDidboroughpresidentproposealternativesite: COMMON_OPTIONSETS.YES_NO_INTEGER,
+    dcpRelatedacquisitionofpropertyopt: LANDUSE_FORM_OPTIONSETS.DCPRELATEDACQUISITIONOFPROPERTYOPT,
```
Comment (godfreyyeung) [inline client/app/helpers/optionset.js:84]: can this just be assigned `COMMON_OPTIONSETS.YES_NO_INTEGER`?

File: client/app/optionsets/landuse-form.js [hunk 4]
```diff
@@ -96,6 +96,28 @@ export const DCPINDICATETYPEOFFACILITY = {
   },
 };
 
+export const DCPRELATEDACQUISITIONOFPROPERTYOPT = {
```
Comment (godfreyyeung) [inline client/app/optionsets/landuse-form.js:99]: hmm this looks just like `COMMON_OPTIONSETS.YES_NO_INTEGER`

**Review by godfreyyeung — APPROVED** [review]

File: client/app/components/packages/landuse-form/change-city-map.hbs (hunk 1 above)
Comment (trbmcginnis) [reply client/app/components/packages/landuse-form/change-city-map.hbs:297]: Yep! lolll the three easement fields in this file are all booleans whereas the other ones are integers (0, 1) -- so messy!

File: client/app/components/packages/landuse-form/change-city-map.hbs (hunk 2 above)
Comment (trbmcginnis) [reply client/app/components/packages/landuse-form/change-city-map.hbs:331]: I've seen other examples of boolean radio buttons where we do it this way, not sure what the reason is though, maybe just outdated code -- if there's not a particular reason for doing it this way, I can create another PR fixing up all these cases.

File: client/app/optionsets/landuse-form.js (hunk 4 above)
Comment (trbmcginnis) [reply client/app/optionsets/landuse-form.js:99]: true!

File: client/app/helpers/optionset.js (hunk 3 above)
Comment (trbmcginnis) [reply client/app/helpers/optionset.js:84]: yep!

**Pull request merged**